org-code-javabuilder/protocol/src/main/java/org/code/protocol/GlobalProtocol.java

```diff
@@ -16,6 +19,7 @@
   private final String dashboardHostname;
   private final String channelId;
   private final AssetFileHelper assetFileHelper;
+  private Set<MessageHandler> messageHandlers;
```

Comment:
Help me understand when we'll have multiple message handler?

Reply:
If we ever need another one for neighborhood, theater, some other new feature in the future. This is mainly here to prevent hard coding playground-specific logic into protocol